config/prod/dgutman-dsa-server-prod-ec2.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-prod-ec2"
```

Comment:
Please rename this to something like 'dgutman-dsa-server-prerprod-ec2`, as its not for prod and I don't want us to be confused about that later.